test/scenarios/cloudnative/codemodules.go

```diff
@@ -39,6 +39,7 @@ const (
 	codeModulesVersion     = "1.246.0.20220627-183412"
 	codeModulesImage       = "quay.io/dynatrace/codemodules:" + codeModulesVersion
 	codeModulesImageDigest = "7ece13a07a20c77a31cc36906a10ebc90bd47970905ee61e8ed491b7f4c5d62f"
+	diskUsageDelta         = 100000
```

Comment:
checked, and `du -c` (`getDiskUsage` uses that) gives you back the size in `Kibibytes`, so a delta of `100000` is ~`100MB` which is fine as a oneagent we download is ~`860M`.

We should still mention in the code that `diskUsageDelta` uses which unit.

maybe `diskUsageKiBDelta` or `diskUsageKibiByteDelta` ?

Reply:
YESS, always add units!

Reply:
good point, I will go with `diskUsageKiBDelta`